src/CSharpFunctionalExtensions.FluentAssertions/ResultExtensions.cs

```diff
@@ -0,0 +1,51 @@
+﻿using FluentAssertions;
+using FluentAssertions.Execution;
+using FluentAssertions.Primitives;
+
+namespace CSharpFunctionalExtensions.FluentAssertions;
+
+public static class ResultExtensions
+{
+    public static ResultAssertions Should(this Result instance) => new(instance);
+}
+
+public class ResultAssertions : ReferenceTypeAssertions<Result, ResultAssertions>
+{
+    public ResultAssertions(Result instance) : base(instance) { }
+
+    protected override string Identifier => "Result";
+
+    /// <summary>
+    /// 
+    /// </summary>
+    /// <param name="because"></param>
+    /// <param name="becauseArgs"></param>
+    /// <returns></returns>
+    public AndConstraint<ResultAssertions> BeSuccessful(string because = "", params object[] becauseArgs)
```

Comment:
Maybe the extension method names should change so they can be consistent with each other

`sut.Should().Succeed()`
`sut.Should().Fail()`

I had some custom assertions that let the caller specify what the success/failure value should be
https://github.com/seangwright/typescript-functional-extensions/blob/main/test/expectExtensions.ts#L50

That might be helpful here.

Reply:
@seangwright Wouldn't that be used in the case of Result<T> not Result? I think that exists in the case of `BeSuccessfulWith(T value)` [here](https://github.com/NitroDevs/CSharpFunctionalExtensions.FluentAssertions/blob/main/src/CSharpFunctionalExtensions.FluentAssertions/ResultTExtensions.cs)

Reply:
Also, I'm okay with simplifying the method names to `Succeed()` and `Fail()`

Reply:
Updated the method names

Reply:
Derp. Forgot that already existed.

Reply:
Just wanted to make sure it wasn't a use case I didn't understand